Misc/NEWS.d/next/Windows/2024-12-05-00-21-11.gh-issue-80192.28PTj4.rst

```diff
@@ -0,0 +1 @@
+Fixed valid ssl certificates being rejected.
```

Comment:
Use a reference:
```suggestion
Fixed valid :mod:`ssl` certificates being rejected.
```

Reply:
Reference added

Reply:
I would see a broader text, e.g. the PR verifies certifications rejected by OpenSSL by additional calls to Windows specific API functions.
The exact proposal text is definitely not fine, but you have got my idea.